src/main/java/group1/intern/controller/customer/PayController.java

```diff
@@ -0,0 +1,30 @@
+package group1.intern.controller.customer;
+
+import group1.intern.bean.PayForm;
+import jakarta.validation.Valid;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
+public class PayController {
```

Comment:
```suggestion
public class PaymentsController {
```
ri luôn cho giống url sếp